extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -421,39 +477,33 @@ public function sendLog() {
 	public function setMediaLinks( $oPage ) {
 		wfProfileIn(__METHOD__);
 
-		$links = array( 'image' => 0, 'video' => 0 );
+		$links = [ self::IMAGE => 0, self::VIDEO => 0 ];
 		if ( isset( $oPage->mPreparedEdit ) && isset( $oPage->mPreparedEdit->output ) ) {
 			$images = $oPage->mPreparedEdit->output->getImages();
 			if ( !empty($images) ) {
 				foreach ($images as $iname => $dummy) {
 					$file = wfFindFile($iname);
 					if ($file instanceof LocalFile) {
 						$mediaType = $file->getMediaType();
-						switch ($mediaType) {
-							case MEDIATYPE_VIDEO:
-								$links['video']++;
-								break;
-							default:
-								$links['image']++;
-						}
+						$linkName = $mediaType === MEDIATYPE_VIDEO ? self::VIDEO : self::IMAGE;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)